server/controllers/userController.js

```diff
@@ -0,0 +1,62 @@
+const db = require('../models/quizModels');
+
+const userController = {};
+
+userController.createUser = (req, res, next) => {
+  const username = req.body.username;
+  const password = req.body.password;
```

Comment:
Can abbreviate lines 6 and 7 to `const { username, password } = req.body`;

Reply:
Agreed, updated.

Reply:
Looks good.